### ISSUE TITLE
Feature/Client - Type answer code to questions

### ISSUE THREAD

**Issue by podiman** [code-mentoring/learn#274]
## The story:
As a **student** that **wants to learn how to code**, I want to **be able to write the correct code**

---

## What we should do:
We should **build a empty space** so that **the student can write the answer to the question**.

---

## To be successful we need to:
- [ ] Create a blank div to write an answer
---

## Design
![image](https://user-images.githubusercontent.com/3611890/87865337-1ecd0a80-c942-11ea-95e1-6975bc230ea5.png)

[LINK TO FIGMA DESIGN](https://www.figma.com/file/eyk5tQgLhIpyiORYfWxeUh/Learning-App?node-id=1546%3A469)

**Comment by ahmedfaaid** [comment]
Hi @podiman is this issue blocked by another issue or can I start working on it?
